db/write_batch.cc

```diff
@@ -2623,10 +2623,11 @@ class MemTableInserter : public WriteBatch::Handler {
       if (!cfd->is_delete_range_supported()) {
         // TODO(ajkr): refactor `SeekToColumnFamily()` so it returns a `Status`.
         ret_status.PermitUncheckedError();
+        InstrumentedMutexLock lock(db_->mutex());
```

Comment:
mutex can already be held if this is called during db open: https://github.com/facebook/rocksdb/blob/a44f4b86530214b1c9d0fccecf8321e2c28663ad/db/db_impl/db_impl_open.cc#L1134-L1335.

Reply:
I think I'll just simplify the message to avoid the complication. Thanks!